spec/controllers/catalog_controller_spec.rb

```diff
@@ -99,6 +99,21 @@
       post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
       expect(response.body).to include(druid_no_pre)
     end
+    context 'can take two forms of druid input' do
+      let(:druid) { 'druid:jj925bx9565' }
+      let(:druid_no_pre) { 'jj925bx9565' }
+
+      it 'druid:jj925bx9565' do
```

Comment:
you can use a variable:  

```ruby
it prefixed_druid do
  blah
end
```

if that doesn't work, try

```ruby
it "#{prefixed_druid}" do
```

Reply:
tried both - didn't work

  `prefixed_druid` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc).